bin/proving-service/src/api/prover.rs

```diff
@@ -60,6 +60,7 @@ impl ProverRpcApi {
         fields(id = tracing::field::Empty),
         err
     )]
+    #[allow(clippy::result_large_err)]
```

Comment:
On latest `main` I don't get any clippy errors for this; where did these come from?

Reply:
Yeah, these start appearing only after I ran `cargo update`.